src/main/java/com/google/sps/servlets/resources/YouTubeServlet.java

```diff
@@ -74,7 +74,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         YouTube youtubeService = getService();
         long results = 5;
         // Use for creating links
-        String templateLink = "https://www.youtube.com/embed/"; //"https://www.youtube.com/watch?v=";
+        String embedTemplateLink = "https://www.youtube.com/embed/";
```

Comment:
Thanks for the explanation of the PR for the differences between the two URLs, would be useful to include a comment here about it too.

Reply:
I can go ahead and do that